### PR TITLE
Add AISWare Flying Server Compatibility certification

### LINKED ISSUE #1470: Compatibility certification request for AISWare Flying Server for Full Platform
To include your Compatible Software Product on the Jakarta EE official list of Compatible Software Products, please provide the following information:

**The Eclipse Foundation profile [Wang Feng](https://accounts.eclipse.org/users/wangfeng) of Hunan AsiaInfo Anhui Technology Limited**

**[AISWare Flying Server V2.0.0](http://asiainfoah.com/#/product/flyingserver)**

**The Profile (Full Platform) that the Compatible Product is submitted to be listed under**
   Specification Name, Version and download URL:
   [Jakarta EE Platform, Full Profile 8](https://jakarta.ee/specifications/platform/8/)  
   TCK Version, digital SHA-256 fingerprint and download URL:
    [Jakarta EE Platform CTS 8.0.2](http://download.eclipse.org/jakartaee/platform/8/eclipse-jakartaeetck-8.0.2.zip) , SHA-256: 14a21b617bb646055c2952f1422ec04a71389fb37301e1c2969f6c3700aee965
    Public URL of TCK Results Summary:
    [AISWare Flying Server 2.0.0 TCK Results](https://github.com/AsiainfoAnhui/aisware_flyingserver_v2_ee8/blob/master/aisware_flyingserver_v2_ee8.md#tck-results)
   Any Additional Specification Certification Requirements:
    Jakarta Dependency Injection 1.0 TCK  
    See test results in section di.  
    Download URL & SHA-256: [jakarta.inject-tck-1.0-bin.zip](https://download.eclipse.org/jakartaee/dependency-injection/1.0/jakarta.inject-tck-1.0-bin.zip), SHA-256: b679bac9b1057df894753892a880ba6ade530607dd811157106ed767aa26481f
    Jakarta Contexts and Dependency Injection 2.0 TCK  
    See test results in section cdi.  
    Download URL & SHA-256: [cdi-tck-2.0.6-dist.zip](https://download.eclipse.org/jakartaee/cdi/2.0/cdi-tck-2.0.6-dist.zip), SHA-256: 75e969a7a3b3c77332154a2008309aad821a923d8684139242048a7640762808
    Jakarta Bean Validation 2.0 TCK,  
    See test results in section beanvalidation.  
[beanvalidation-tck-dist-2.0.5.zip](https://download.eclipse.org/jakartaee/bean-validation/2.0/beanvalidation-tck-dist-2.0.5.zip), SHA-256: b6778914f78bfcce5d6934347d71502603b1b0a6bbfdfbcf956271c367d40974
    Jakarta Batch 1.0 TCK tests are included in Jakarta EE 8 CTS tests, See test results in section jbatch  
    Java SE Provider & Version tested with:
        OpenJDK Runtime Environment (build 25.262-b02, mixed mode)
    Summary of the information for the certification environment:
        Apache Derby 10.14.2.0, Linux, 4.19.90-17.ky10.aarch64
        Interoperatibility tests carried out against Eclipse Glassfish 5.1.0

**[AISWare FlyingServer Logo](https://github.com/AsiainfoAnhui/aisware_flyingserver_v2_ee8/blob/master/AISWare_FlyingServer_Logo.zip)**

**Hunan AsiaInfo Anhui Technology Limited**

**[Compatibility certification request for [AISWare Flying Server] for [Full Platform] #487](https://github.com/eclipse-ee4j/jakartaee-platform/issues/487)**

**[AISWare FlyingServer Home Page](https://cms.101test.com/ai_sino_info/zh_cn/content_603710660.html)**

### PR DESCRIPTION
**Pull request by ericpoirier**
Fixes #1470

Signed-off-by: Eric Poirier <eric.poirier@eclipse-foundation.org>

### CODE REVIEW

**Comment by ericpoirier** [comment]
We have  the +1 from Tanja.

**Review by linkfang — CHANGES_REQUESTED** [review]
LGTM! But as the issue creator mentioned in the issue https://github.com/jakartaee/jakarta.ee/issues/1470#issuecomment-1125605258, he updated the logo and will update the download link. So, I think we will need to wait for his update before merging.

**Comment by ericpoirier** [comment]
> LGTM! But as the issue creator mentioned in the issue [#1470 (comment)](https://github.com/jakartaee/jakarta.ee/issues/1470#issuecomment-1125605258), he updated the logo and will update the download link. So, I think we will need to wait for his update before merging.

Yes, thanks for clarifying this on the PR. I was waiting for the updated link to update this PR.

**Comment by ericpoirier** [comment]
The logo and link has been updated.

We can review and merge now.

**Comment by linkfang** [comment]
Cool! But found another small issue. The 2.0.0 link under Download Compatible Products - Jakarta 8 will download a Jakarta zip file instead of the product itself. Most items under 9.1 will download the product directly or take users to the download page. Like,  Eclipse GlassFish and Apusic AAS etc.

File: data/compatible_products.yml
Comment (linkfang) [inline data/compatible_products.yml]: I think this is causing the issue I talked about. Looks like a lot of items here under 8.0 are using the same url for both compatibility and download_url. But I think the download_url should be the same as the one for download?

Comment (linkfang) [reply data/compatible_products.yml]: And the compatibility should be link to the tck result link?

Comment (ericpoirier) [reply data/compatible_products.yml]: The Compatibility link should go to the TCK results page which is:

https://github.com/AsiainfoAnhui/aisware_flyingserver_v2_ee8/blob/master/aisware_flyingserver_v2_ee8.md#tck-results

But as for the download_url link, there doesn't seem to be a strict standard as to what to put there if I look at other Compatible products. 

Looking at Compatible products from Jakarta EE 9 and 9.1, most of them have a zip file as their download_url. That's what i'm going to use.

But yeah it's either a zip file or the same link as the Download link.

Comment (linkfang) [reply data/compatible_products.yml]: > 
> Looking at Compatible products from Jakarta EE 9 and 9.1, most of them have a zip file as their download_url. That's what i'm going to use.

Yep, the product zip file will work for sure (e.g. Eclipse GlassFish 6.1). 
But the zip file you have for the download_url here is not the product zip file but a jakartaeetck zip file. So, I think it would be better if we use either the product zip file or the download link if it's not available/provided.

**Review by linkfang — APPROVED** [review]

File: data/compatible_products.yml
Comment (ericpoirier) [reply data/compatible_products.yml]: Ok I see, I put the same url as for the Download link since it was not provided.

Comment (linkfang) [reply data/compatible_products.yml]: Cool!

**Pull request merged**
